architecture-decision-record/0028-expand-the-scope-of-the-platform.md

```diff
@@ -0,0 +1,77 @@
+# 28. Expand the Scope of the Platform
+
+Date: 2023-11-14
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Modernisation Platform was originally developed with the intention that it would only be used for migrating legacy applications to a strategic cloud platform, where they could not be hosted in the [Cloud Platform](https://github.com/ministryofjustice/cloud-platform).  Applications would then be hosted on the Modernisation Platform until they were either modernised and moved to the Cloud Platform, or decommissioned or replaced with newer applications.
```

Comment:
> until they were either modernised and moved to the Cloud Platform, or decommissioned or replaced with newer applications.

Maybe instead put this as:

> until they were modernised and moved to the Cloud Platform, replaced with newer applications, or decommissioned.

That way, it removes “either” (there are three options), and goes from most to least favourable.